awx/ui/client/features/templates/list.view.html

```diff
@@ -0,0 +1,117 @@
+<div ui-view="form"></div>
+<at-panel>
+    <at-panel-heading hide-dismiss="true">
+        {{:: vm.strings.get('list.PANEL_TITLE') }}
+        <div class="at-Panel-headingTitleBadge" ng-show="template_dataset.count">
+            {{ template_dataset.count }}
+        </div>
+    </at-panel-heading>
+
+    <at-panel-body>
+        <div class="at-List-toolbar">
+            <smart-search
+                class="at-List-search"
+                django-model="unified_job_templates"
+                base-path="unified_job_templates"
+                iterator="template"
+                list="list"
+                dataset="template_dataset"
+                collection="collection"
+                search-tags="searchTags"
+                query-set="querySet">
+            </smart-search>
+            <div class="at-List-toolbarAction" ng-show="canAdd">
+                <button
+                    type="button"
+                    class="at-List-toolbarActionButton at-Button--success"
+                    data-toggle="dropdown"
+                    aria-haspopup="true"
+                    aria-expanded="false">
+                        &#43; {{:: vm.strings.get('list.ADD_BUTTON_LABEL') }}
+                        <span class="at-List-toolbarDropdownCarat"></span>
+                        <span class="sr-only">Toggle Dropdown</span>
+                </button>
+                <ul class="dropdown-menu at-List-toolbarActionDropdownMenu">
+                    <li>
+                        <a ui-sref="templates.addJobTemplate">
+                            {{:: vm.strings.get('list.ADD_DD_JT_LABEL') }}
+                        </a>
+                    </li>
+                    <li>
+                        <a ui-sref="templates.addWorkflowJobTemplate">
+                            {{:: vm.strings.get('list.ADD_DD_WF_LABEL') }}
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <at-list results="templates">
+            <!-- TODO: implement resources are missing red indicator as present in mockup -->
+            <at-row ng-repeat="template in templates"
+                ng-class="{'at-Row--active': (template.id === vm.activeId)}">
+                <div class="at-Row-items">
+                    <at-row-item
+                        header-value="{{ template.name }}"
+                        header-link="/#/templates/job_template/{{ template.id }}"
+                        header-tag="{{ vm.templateTypes[template.type] }}"
+                        ng-if="template.type === 'job_template'">
+                    </at-row-item>
+                    <at-row-item
+                        header-value="{{ template.name }}"
+                        header-link="/#/templates/workflow/{{ template.id }}"
```

Comment:
Change link to _header-link="/#/templates/workflow_job_template/{{ template.id }}"_